CHANGELOG.rst

```diff
@@ -7,6 +7,7 @@ master (unreleased)
 
 - Added tests for the validators mapping
 - Minor syntax changes
+- Add configuration for py.test
```

Comment:
you can also add "reactivated accidentally skipped `test_validations.py` tests"

Reply:
Added.